docs/source/tools_integration/ipython.md

```diff
@@ -43,8 +43,8 @@ In [2]: %reload_kedro <path_to_project_root>
 In [3]: %reload_kedro
 ```
 
-```eval_rst
-.. note:: If you want to pass arguments to the :code:`reload_kedro` line magic, e.g. to specify a configuration environment, you should call it as a normal Python function (:code:`reload_kedro(env=env, extra_params=extra_params`) rather than using :code:`%reload_kedro` in a Notebook cell (:code:`%reload_kedro(extra_params=extra_params)` wouldn’t work). You might have to call :code:`%automagic False` beforehand to make this work.
+```{note}
+If you want to pass arguments to the :code:`reload_kedro` line magic, e.g. to specify a configuration environment, you should call it as a normal Python function (:code:`reload_kedro(env=env, extra_params=extra_params`) rather than using :code:`%reload_kedro` in a Notebook cell (:code:`%reload_kedro(extra_params=extra_params)` wouldn’t work). You might have to call :code:`%automagic False` beforehand to make this work.
```

Comment:
```suggestion
If you want to pass arguments to the `reload_kedro` line magic, e.g. to specify a configuration environment, you should call it as a normal Python function (`reload_kedro(env=env, extra_params=extra_params)`) rather than using `%reload_kedro` in a Notebook cell (`%reload_kedro(extra_params=extra_params)` wouldn't work). You might have to call `%automagic False` beforehand to make this work.
```